.travis.yml

```diff
@@ -1,9 +1,17 @@
-language: bash
-before_script:
-  - "sudo apt-get install zsh"
+language: objective-c # force OS X
+before_install:
+  - brew update
+  - brew config
+install:
+  - mkdir -p $(brew --repo)/Library/Taps/homebrew
+  - ln -s $PWD $(brew --repo)/Library/Taps/homebrew/homebrew-command-not-found
```

Comment:
No, we can't. We need to test current version instead of master version.

FYI, I haven't get test work yet.